setup.py

```diff
@@ -0,0 +1,51 @@
+from setuptools import find_packages, setup
+
+# This allows us to install the package outside of the repository using the following command:
+# pip install git+https://github.com/ministryofjustice/operations-engineering.git
+setup(
```

Comment:
Does this mean we need to maintain 2 package version lists going forward? Here in `setup.py` and in `Pipfile`? Is there a neater way of doing this so we don't repeat? Follow up - why is this slightly different to what's in the `Pipfile`?